tox.ini

```diff
@@ -202,19 +202,22 @@ commands = pytest -v tests/test_wsgi_servers.py
 # --------------------------------------------------------------------
 
 [smoke-test]
-commands = pip uninstall --yes msgpack
+# NOTE(vytas): Work around pprofile's reliance on the removed use_2to3 option:
+#   https://github.com/vpelletier/pprofile/issues/41
+commands = pip install "setuptools < 58.0"
+           pip install -r {toxinidir}/requirements/bench
            python -c "import falcon.asgi; falcon.asgi.App()"
            falcon-bench -t 1 -b falcon-ext
 
 [testenv:py38_smoke]
 basepython = python3.8
-deps = -r{toxinidir}/requirements/bench
+deps =
 commands = {[smoke-test]commands}
 
 [testenv:py38_smoke_cython]
 basepython = python3.8
-deps = -r{toxinidir}/requirements/bench
-       cython
+deps = cython
+setenv = {[with-cython]setenv}
```

Comment:
I believe that the cythonized smoke test was not working as intended before this change.